aws_lambda_powertools/shared/functions.py

```diff
@@ -1,7 +1,16 @@
-from distutils.util import strtobool
 from typing import Any, Optional, Union
 
 
+def _strtobool(value):
```

Comment:
As we're removing distutil package and this will be deprecated, it's okay to keep the same name (no `_`)